scripts/tests/units/delayPriceFeed/DelayFathomOracleDex.test.js

```diff
@@ -0,0 +1,349 @@
+const chai = require('chai');
+const { expect } = chai
+const { solidity } = require("ethereum-waffle");
+chai.use(solidity);
+const { BigNumber, ethers } = require("ethers");
+const { WeiPerRad, WeiPerRay, WeiPerWad } = require("../../helper/unit");
+const { DeployerAddress, AliceAddress, BobAddress } = require("../../helper/address");
+const { getContract, createMock, connectToContractWithAddress } = require("../../helper/contracts");
+const { increase } = require('../../helper/time');
+const { weiToRay, weiToDecimal, rayToDecimal } = require('../../helper/unit');
+const { formatBytes32String } = ethers.utils
+const { getDeadlineTimestamp } = require("../../helper/timeStamp");
+const { approve } = require("../../helper/token");
+
+describe("Delay Fathom Oracle with DexPriceOracle - Unit Test Suite", () => {
+    let dexPriceOracle;
+    let delayFathomOraclePriceFeed;
+    let mockPriceOracle;
+
+    let bookKeeper;
+    let collateralPoolConfig;
+    let accessControlConfig;
+
+    let mockedBookKeeper;
+    let mockedCollateralPoolConfig;
+
+    let Router;
+
+    let dexToken0 = "0xce75A95160D96F5388437993aB5825F322426E04";
+    let dexToken1 = "0x0D2B0406bc8400E61f7507bDed415c98E54A8b11";
+    let dexFactoryAddress = "0xb9AdA6B44E4CFF8FE00443Fadf8ad006CfCc2d10";
+    let routerAddress = "0xf72f1a39ae0736Ef6A532605C85aFB0A4E349714";
```

Comment:
Hardcoded address of contract that is not part of migration will not work for tests run